daemons/compute/server/servers/server_default.go

```diff
@@ -465,13 +652,39 @@ func (s *defaultServer) Delete(ctx context.Context, req *pb.DataMovementDeleteRe
 	}, nil
 }
 
-func (s *defaultServer) findRabbitRelativeSource(ctx context.Context, req *pb.DataMovementCreateRequest) (string, error) {
+func (s *defaultServer) findDestinationLustreFilesystem(ctx context.Context, dest string) (*lusv1alpha1.LustreFileSystem, error) {
 
-	computeMountInfo, err := s.findComputeMountInfo(ctx, req)
-	if err != nil {
-		return "", err
+	if !filepath.IsAbs(dest) {
+		return nil, fmt.Errorf("Destination must be an absolute path")
+	}
+	origDest := dest
+	if strings.TrimRight(dest, "/") == dest {
```

Comment:
This is a little strange to me. To me it would make more sense if you said
```go
if !strings.HasSuffix(dest, "/") {
    dest += "/"
}
 ```

Reply:
Should we be doing this at all? If the lustre fs mount root is "/mnt/global" and they provide dest "/mnt/global" - that's not a valid request. Dest should be at the very least "/mnt/global/"

Reply:
This is a place  where I want to distinguish the user specifying a dest of "/lus/foosball" and not having the LustreFileSystem's "/lus/foo" count as a matching prefix.
So I append a "/" to both when checking for a match.
Beyond that, I haven't thought about what sort of dest should or shouldn't be allowed.

Reply:
Ohh good point - I didn't see that use case.  Still prefer the HasSuffix